### PR TITLE
improv/SDPOSTE-52 MenigaTransaction fetch optional inlude request

### PR DESCRIPTION
**Pull request by agustk**
Adding include to the filter.

### CODE REVIEW

File: sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java [hunk 1]
```diff
@@ -252,6 +266,20 @@ protected TransactionsFilter clone() throws CloneNotSupportedException {
 		return map;
 	}
 
+	private String constructInclude() {
```
Comment (arturdm) [inline sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java]: There's `?include=Account,Merchant` in `getTransactions` method in `MenigaAPI.java`. I guess it's not needed anymore.

File: sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java [hunk 2]
```diff
@@ -231,7 +241,11 @@ protected TransactionsFilter clone() throws CloneNotSupportedException {
 		Field skipField = null;
 		Field takeField = null;
 		for (Field member : mappableFields) {
-			if (member.getName().equals("isFiltering")) {
+			if (member.getName().equals("isFiltering") || member.getName().equals("includeMerchants")) {
```
Comment (arturdm) [inline sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java]: Shouldn't `member.getName().equals("includeMerchants")` go to the check below?

File: sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java (hunk 1 above)
Comment (agustk) [reply sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java]: Oh yes, need to remove that. I was wondering how they were getting included.

File: sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java (hunk 2 above)
Comment (agustk) [reply sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java]: No, we'd get "include=xxx" twice then.

**Pull request merged**